### PR TITLE
Add i18n for organizations views

### PR DESCRIPTION
**Pull request by CGA1123**
Hi, again another PR related to #662.

### CODE REVIEW

File: app/views/organizations/_reset_and_remove_organization_modal.html.erb [hunk 1]
```diff
@@ -1,24 +1,23 @@
...
   <div class="remodal-warning">
-    Unexpected things will happen if you don't read this!
+    <%= t('views.organizations.delete_warning') %>
   </div>
 
   <div class="remodal-description">
     <p>
-      This action <strong>CANNOT</strong> be undone.
-      Please note that reseting and removing this classroom will delete all repositories and teams under the <strong><%= @organization.github_organization.login %></strong> account that were created by GitHub Classroom.
+      <%= t('views.organizations.delete_concequences', organization: @organization.github_organization.login).html_safe %>
```
Comment (cyhsutw) [inline app/views/organizations/_reset_and_remove_organization_modal.html.erb]: Hi @CGA1123 👋 

Thanks for the pull requests. 

One tiny nit here: `concequences` is misspelled, it should be `consequences`. 

Also, don't forget to update the [`config/locales/views/organizations/en.yml`](https://github.com/education/classroom/pull/739/files#diff-7cab1873e635f29b4f267aa5665ced5bR27) 😉

Comment (CGA1123) [inline app/views/organizations/_reset_and_remove_organization_modal.html.erb]: Hey @cyhsutw, 
Thanks for spotting that! Should be fixed now.

**Comment by nwoodthorpe** [comment]
This is still valid, we just haven't had time to review it.

**Comment by CGA1123** [comment]
@nwoodthorpe I can rebase this onto the latest master if you want.

**Comment by nwoodthorpe** [comment]
Please do! @CGA1123

**Comment by CGA1123** [comment]
@nwoodthorpe Sorry for delay, was at work. Branch is now rebased on latest master.

**Comment by nwoodthorpe** [comment]
Thanks! This is a big one, I'll try to get to it soon.

**Comment by CGA1123** [comment]
No problem!

Sent from my iPhone

On 13 Jul 2017, at 21:11, Nathaniel Woodthorpe <notifications@github.com<mailto:notifications@github.com>> wrote:


Thanks! This is a big one, I'll try to get to it soon.

File: app/views/organizations/show.html.erb [hunk 2]
```diff
@@ -7,7 +7,7 @@
 
         <div class="col-6 float-right text-right pl-2">
           <% if @assignments.present? %>
-            <%= link_to 'New assignment', new_assignment_organization_path(@organization), class: 'btn btn-primary right', role: 'button'%>
```
Comment (nwoodthorpe) [inline app/views/organizations/show.html.erb]: The translation here has an upper case A, should be lowercase

**Comment by CGA1123** [comment]
@nwoodthorpe I think these latest commits should fix up all the issues you brought up. Let me know if not.

**Comment by nwoodthorpe** [comment]
This lgtm!

**Comment by nwoodthorpe** [comment]
Welp, CI isn't cooperating right now.

ran `script/linter` and `rspec` locally, all :+1:

**Pull request merged**